client/my-sites/plugins/index.js

```diff
@@ -35,6 +36,16 @@ module.exports = function() {
 	}
 
 	if ( config.isEnabled( 'manage/plugins' ) ) {
+		page( '/plugins/wpcom-masterbar-redirect/:site', context => {
```

Comment:
Is this cleaner than using `context.query` in a similar way? https://github.com/Automattic/wp-calypso/blob/master/client/boot/common.js#L52 I don't have a strong opinion (feel free to say "yes it is")... query parameters are ugly, but they could be ephemeral if we redirected to the url without the param.